$OpenBSD: patch-platform_gl_gl-main_c,v 1.1 2016/01/19 05:20:51 semarie Exp $

add pledge(2) to mupdf-gl:
  - drm : opengl stuff
  - proc exec : opening a external browser on uri link

--- platform/gl/gl-main.c.orig	Tue Nov 10 17:19:51 2015
+++ platform/gl/gl-main.c	Sun Jan 17 10:21:44 2016
@@ -1361,6 +1361,12 @@ int main(int argc, char **argv)
 
 	glfwMakeContextCurrent(window);
 
+	if (pledge("stdio rpath drm proc exec", NULL) == -1)
+	{
+		fprintf(stderr, "pledge: %s\n", strerror(errno));
+		exit(1);
+	}
+
 	ctx = fz_new_context(NULL, NULL, 0);
 	fz_register_document_handlers(ctx);
 
